.gitignore

```diff
@@ -26,6 +26,7 @@ bob_robotics.includes
 
 # Ignore object files
 **/*.a
+**/*.so
```

Comment:
For symmetry should we also be ignoring .dll?

Reply:
I *think* we already are